contrib/managed-azure/create_basic_hosted_cluster.sh

```diff
@@ -79,47 +104,40 @@ az identity federated-credential create --name "${CSI_FILE_CCM_MSI_NAME}"-fed-id
 --subject system:serviceaccount:openshift-cluster-csi-drivers:azure-file-csi-driver-node-sa \
 --audience openshift
 
-az identity federated-credential create --name "${IR_MSI_NAME}"-fed-id \
---identity-name "${IR_MSI_NAME}" \
---resource-group "${PERSISTENT_RG_NAME}" \
---issuer "${OIDC_ISSUER_URL}" \
---subject system:serviceaccount:openshift-image-registry:cluster-image-registry-operator \
---audience openshift
-
 az identity federated-credential create --name "${IR_MSI_NAME}"-fed-id \
 --identity-name "${IR_MSI_NAME}" \
 --resource-group "${PERSISTENT_RG_NAME}" \
 --issuer "${OIDC_ISSUER_URL}" \
 --subject system:serviceaccount:openshift-image-registry:registry \
 --audience openshift
 
+# Create the HostedCluster; leave off --control-plane-operator-image line if you aren't testing with a particular CPO image
 ${HYPERSHIFT_BINARY_PATH}/hypershift create cluster azure \
 --name "$CLUSTER_NAME" \
 --azure-creds $AZURE_CREDS \
 --location ${LOCATION} \
 --node-pool-replicas 2 \
---base-domain $AZURE_BASE_DOMAIN \
+--base-domain $PARENT_DNS_ZONE \
 --pull-secret $PULL_SECRET \
 --generate-ssh \
 --release-image ${RELEASE_IMAGE} \
---external-dns-domain ${MGMT_DNS_ZONE_NAME} \
+--external-dns-domain ${DNS_ZONE_NAME} \
 --resource-group-name "${MANAGED_RG_NAME}" \
 --vnet-id "${GetVnetID}" \
 --subnet-id "${GetSubnetID}" \
 --network-security-group-id "${GetNsgID}" \
 --sa-token-issuer-private-key-path "${SA_TOKEN_ISSUER_PRIVATE_KEY_PATH}" \
 --oidc-issuer-url "${OIDC_ISSUER_URL}" \
 --annotations hypershift.openshift.io/pod-security-admission-label-override=baseline \
---control-plane-operator-image=<YOUR_REPO>:"${TAG}" \
---managed-identities-file ${MANAGED_IDENTITIES_FILE} \
+--control-plane-operator-image=quay.io/user/hypershift:"${TAG}" \
 --fips=true \
 --marketplace-publisher azureopenshift \
 --marketplace-offer aro4 \
 --marketplace-sku aro_417 \
 --marketplace-version 417.94.20240701 \
 --dns-zone-rg-name os4-common \
 --assign-service-principal-roles \
---managed-identities-file /Users/your username/aro-hcp-control-plane-MIs.json \
---data-plane-identities-file /Users/your username/aro-hcp-data-plane-MIs.json
+--managed-identities-file /Users/user/aro-hcp-control-plane-MIs.json \
+--data-plane-identities-file /Users/user/aro-hcp-data-plane-MIs.json
```

Comment:
Should we also add the diagnostics flag? It let's you see the serial console on the VM.